### ISSUE TITLE
PS2 Support

### ISSUE THREAD

**Issue by YAGAMI55** [VelocityRa/SlyTools#4]
Hi. how can i extract wal/wac files from ps2 iso?
All of them are hidden. ps2 iso is a raw disc like Ratchet and clank

**Comment by VelocityRa** [comment]
No, PS2 is not supported. I don't plan to work on it. The PS3 assets are higher quality anyway, so I they're better for my use cases.

**Comment by YAGAMI55** [comment]
Understand) Anyway. thx for you work and tools)

**Comment by VelocityRa** [comment]
We can leave this open in case I or someone else decides to pick it up.

I've already spent a bit of time on this so if anyone is interested in working on it, let me know and I'll get you up to speed on what you'd have to do to implement it.

**Comment by YAGAMI55** [comment]
So. main trouble is get files from .iso because they hidden. .iso is a raw disc format like Rathet and Clank. but in rac table of content placed in some .irx files. Maybe in ps2 version of Sly something like that

**Comment by VelocityRa** [comment]
@YAGAMI55 Interesting. The TOC for Sly is encrypted and it's placed on a hardcoded address in the .iso, which I don't think is in any of the .irxes.
Join the Sly Modding Discord if you want to work on this, we have a Ghidra server you can look at where I have marked some relevant things.

**Comment by YAGAMI55** [comment]
> @VelocityRa Sory. not module .irx. In RAC toc in iorp.img mayby sly have same toc.
